src/dbnode/storage/bootstrap/bootstrapper/commitlog/source.go

```diff
@@ -146,6 +146,7 @@ func newCommitLogSource(
 func (s *commitLogSource) AvailableData(
 	ns namespace.Metadata,
 	shardsTimeRanges result.ShardTimeRanges,
+	_ bootstrap.RunState,
```

Comment:
Can probably revert this interface change, doesn't look like we use `RunState` in any `AvailableData`  or `AvailableIndex` implementations